readme.md

```diff
@@ -6,18 +6,85 @@
 [!["help wanted" issues](https://img.shields.io/github/issues/xmldom/xmldom/help%20wanted?color=darkgreen&style=flat-square)](https://github.com/xmldom/xmldom/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
 [![Mutation report](https://camo.githubusercontent.com/ee312c4ebce7784ce9f785757eba5d6e33e6d950/68747470733a2f2f696d672e736869656c64732e696f2f656e64706f696e743f7374796c653d666c61742675726c3d687474707325334125324625324662616467652d6170692e737472796b65722d6d757461746f722e696f2532466769746875622e636f6d25324662726f647962697473253246786d6c646f6d2532466d6173746572)](https://dashboard.stryker-mutator.io/reports/github.com/brodybits/xmldom/master)
 
-A JavaScript implementation of W3C DOM for Node.js, Rhino and the browser. Fully
-compatible with `W3C DOM level2`; and some compatible with `level3`. Supports
-`DOMParser` and `XMLSerializer` interface such as in browser.
+xmldom is a javascript [ponyfill](https://ponyfill.com/) for the following APIs supported in browsers:
+- convert an XML string into a DOM tree (`new DOMParser().parseFromString(xml, mimeType)` => `Document`)
+- create, access and modify a DOM tree (`new DOMImplementation().createDocument(...)` => `Document`, )
+- serialize a DOM tree back into an XML string (`new XMLSerializer().serializeToString(node)` => `string`)
+
+If your code always/only needs to work in a browser, you should rather not use xmldom. But you should (potentially always) use it, when your code needs to also work in other runtimes like NodeJS or Rhino.
+
+The implementation is based on several specs:
+
+## Specs
+
+![Related specifications](docs/specs.svg)
```

Comment:
nit: I don't know how accessible embedding this SVG would be to the visually impaired.

My understanding is that the visually impaired are able to read many websites with help from tools such as text-to-speech.

And another understanding is that at least in some parts of North America, free websites need to be accessible since they are considered public resources, and some less-abled people can sue website owners for websites that are not accessible enough.

Reply:
I would assume that an SVG is more accessible than a PNG file, since it contains the text elements in a semantic manner.
I believe the related PUML file might even be less accessible because of it's weird syntax.
But I also have no clue if there would be a way to list this information in a textual manner including the relationships...

Of course I have no clue about the north american regulations.

Any recommendation?

I think that this is in any way a good starting point since it allows easier access to the related specs, even though not to everyone. Once we discussed which ones we consider relevant for our development we should be able to come up with a shortened textual representation with links to those relevant specs.

Reply:
> I think that this is in any way a good starting point

👍 agreed on my part